doc/tutorials/cpu_sharing.rst

```diff
@@ -37,21 +37,29 @@ Scheduling initialization is invoked in the hardware management layer.
 .. figure:: images/cpu_sharing_api.png
    :align: center
 
-vCPU affinity
+CPU affinity
 *************
 
-Currently, we do not support vCPU migration; the assignment of vCPU
-mapping to pCPU is statically configured by acrn-dm through
-``--cpu_affinity``. Use these rules to configure the vCPU affinity:
-
-- Only one bit can be set for each affinity item of vCPU.
-- vCPUs in the same VM cannot be assigned to the same pCPU.
+Currently, we do not support vCPU migration; the assignment of vCPU mapping to
+pCPU is fixed at the time the VM is launched. The statically configured
+cpu_affinity in the VM configuration defines a superset of pCPUs that
+the VM is allowed to run on. One bit in this bitmap indicates that one pCPU
+could be assigned to this VM, and the bit number is the pCPU ID. A pre-launched
+VM is supposed to be launched on exact number of pCPUs that are assigned in
```

Comment:
... on the exact...?

Reply:
Yes, but "cpu_affinity" in VM configuration is for master, "cpu_affinity_bitmap" is for release_1.6.1.